source/default/_patterns/02-molecules/utchero/_utchero.twig

```diff
@@ -0,0 +1,100 @@
+	{% else %}
+		{% set title_color = 'text-utc-new-blue-500' %}
+		{% set body_color = 'text-base' %}
+		{% set btn_border_color = "lg:border-utc-new-blue-500" %}
+		{% set bg_color = 'background-color:#104dd4' %}
+		{% set bg1_opacity = 'opacity-20' %}
+    {% set bg2_opacity = 'opacity-20' %}
+	{% endif %}
+
+<div class="{{ card_classes }}">
+	{% block utchero_content %}
+  {# image on left #}	 
+	{% if utchero.hero_align == 'Left' %}
+		<div class="lg:grid lg:grid-rows-utchero lg:grid-cols-utchero lg:gap-y-4 lg:grid-flow-row lg:mb-4 text-center">
+    {# TOdo ADD conditional statement if the image contains patternlab data or not. #}
+			{% if utc_hero.hero_image %}
+				<div class="lg:col-start-2 lg:col-end-3 lg:row-start-2 lg:row-end-6 lg:m-auto col-span-1 z-30">
+					{# {{ utchero.hero_image }} #}
+        {% include '@atoms/image/_image.twig' with {
+              image: utchero.hero_image,
+              image_classes: ['w-full']
+        } %}
+          {# <img src='https://via.placeholder.com/1260x840.jpeg'> #}
+				</div>
+			{% endif %}
+				<div class="lg:col-start-3 lg:col-end-4 lg:row-start-1 lg:row-end-4 lg:my-auto lg:ml-8 p-4 z-30">
+	{% else %}
+	{# image on right #}	
+		<div class="lg:grid lg:grid-rows-utchero lg:grid-cols-utcheroright lg:gap-y-6 grid-flow-row lg:mb-4 text-center">
+			{# {% if utc_hero.hero_image %} #}
+				<div class="lg:col-start-3 lg:col-end-4 lg:row-start-2 lg:row-end-5 lg:m-auto z-30">
+					{# {{ utchero.hero_image }} #}
+          {# <img src='https://via.placeholder.com/1260x840.jpeg'> #}
+				</div>
+			{# {% endif %} #}
+				<div class="lg:col-start-2 lg:col-end-2 lg:row-start-1 lg:row-end-4 lg:mr-8 lg:my-auto py-4 px-6 col-span-1 z-30">
+	{% endif %}
+				{% block herocontent %}
+					{% block herotagtitle %}
+						{% if utchero.hero_tag %}
+							<p class="text-left font-semibold tracking-wide uppercase text-lg {{ body_color }}">
+								{{ utchero.hero_tag }}
+							</p>
+							<hr class="border-t-3 border-utc-new-gold-500 mt-2 mb-6 w-20"/>
+						{% endif %}
+						{% if utchero.hero_title %}
+							<h2 class="text-left {{ title_color }}">
+								{{ utchero.hero_title }}
+							</h2>
+						{% endif %}
+					{% endblock herotagtitle %}
+					{% block herotextbutton %}
+						{% if utchero.hero_text %}
+							<p class="text-left text-lg text-base {{ body_color }}">
+								{{ utchero.hero_text}}
+							</p>
+						{% endif %}
+						{% if utchero.hero_button_url %}
+							{% if utchero.hero_button_text %}
+								<p class="my-6 text-right">
+									<a href="{{ utchero.hero_button_url}}" class="font-semibold transition duration-500 ease-in-out {{ body_color }} hover:bg-gray-200 lg:hover:text-utc-new-blue-500 lg:hover:bg-white transform hover:scale-105 py-2 px-4 border {{ btn_border_color }} border-utc-new-blue-500" aria-label="Read the release">
+										Learn more
+										<i class="fas fa-angle-right text-utc-new-gold-500 font-semibold ml-1"></i>
```

Comment:
So, this could eventually be a button type "atom" in Particle that is embedded here, right?

Reply:
that is correct. That's the plan eventually but since we have to get buy-in from the committees and Stephen we are doing two of the biggest components cards and the hero and then splicing them.